src/main/java/LittleMissHelpful/Command/AddEventCommand.java

```diff
@@ -0,0 +1,70 @@
+package LittleMissHelpful.Command;
+
+import java.time.LocalDateTime;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+import LittleMissHelpful.Storage;
+import LittleMissHelpful.TaskList;
+import LittleMissHelpful.Ui;
+import LittleMissHelpful.Exception.InvalidCommandException;
+import LittleMissHelpful.Task.Event;
+
```

Comment:
The commands naming is named well! Easy to understand what the class is handling!